src/main/java/io/lettuce/core/RedisJsonCommandBuilder.java

```diff
@@ -256,6 +353,32 @@ Command<K, V, String> jsonSet(K key, JsonPath jsonPath, JsonValue value, JsonSet
         return createCommand(JSON_SET, new StatusOutput<>(codec), args);
     }
 
+    /**
+     * JSON.SET command with raw string value to avoid unnecessary conversions.
+     *
+     * @param key      the key
```

Comment:
maybe we can just use 1 space here? can you check comment on other files?
```suggestion
     * @param key the key
```

Reply:
Thanks for the review!

First, I'll remove the `@param` documentation from the `RedisJsonCommandBuilder` methods since I noticed that other methods in this class don't have JavaDoc comments - they follow the pattern of keeping the documentation only in the interface level.

Second, I checked the other files where I added `@param` documentation and they already follow the correct single space pattern you mentioned.